cmd/config/internal/commands/cmdcreatesubstitution_test.go

```diff
@@ -29,8 +29,7 @@ func TestCreateSubstitutionCommand(t *testing.T) {
 		{
 			name: "substitution replicas",
 			args: []string{
-				"image", "nginx:1.7.9", "--pattern", "IMAGE:TAG",
-				"--value", "IMAGE=image", "--value", "TAG=tag"},
+				"my-image-subst", "--field-value", "nginx:1.7.9", "--pattern", "${my-image-setter}:${my-tag-setter}"},
```

Comment:
@yuwenma With this change, creating substitutions will be simplified significantly. You will have to input field-value and pattern and skip the rest.

Reply:
Sweet!